pkg/pipelines/tekton/pipelines_pac_provider.go

```diff
@@ -199,7 +204,7 @@ func (pp *PipelinesProvider) createClusterPACResources(ctx context.Context, f fn
 func (pp *PipelinesProvider) createRemotePACResources(ctx context.Context, f fn.Function, metadata pipelines.PacMetadata) error {
 
 	// figure out pac installation namespace
-	installed, installationNS, err := pac.DetectPACInstallation(ctx, "")
+	installed, installationNS, err := pac.DetectPACInstallation(ctx, "openshift-pipelines")
```

Comment:
rationale: this gives the function hint where to search the PaC info config map so all namespace scan is not required (unprivileged user cannot scan all the namespaces)